Gulpfile.js

```diff
@@ -136,6 +137,7 @@ var CLIENT_TESTS_SAUCELABS_SETTINGS = {
     timeout:   720
 };
 
+var websiteServer;
```

Comment:
oh, i forgot. We always initialize a variable with a default value. It's `null` in this case